frontend/src/framework/components/Content/private-components/ViewWrapper/viewWrapper.tsx

```diff
@@ -19,11 +19,11 @@ type ViewWrapperProps = {
     height: number;
     x: number;
     y: number;
-    isDragged: boolean;
-    dragPosition: Point;
```

Comment:
What is the rerender benefit of uniting these two props?

Reply:
We can keep them separate wrt. re-rerendering I think 👍